File: Python/ВУЗ/В.У.З_0.8.py
print("Введите уравнение")
num1, word, num2 = input().split()

if (word == "-"):
    print(float(num1) - float(num2))
elif (word == "+"):
    print(float(num1) + float(num2))
elif (word == "/"):
    print(float(num1) / float(num2))
elif (word == "*"):
    print(float(num1) * float(num2))
else:
    print("Введины ошибочные данные")
input()

File: C++/В.У.З_0.8/В.У.З_0.8.cpp
#include <iostream>

using namespace std;

int main() {
	setlocale(LC_ALL, "RUS");

	double num1, num2;
	char word;

	cout << "Введите пример" << endl;
	cin >> num1 >> word >> num2;

	if (word == '-')
		cout << num1 - num2;
	else if (word == '+')
		cout << num1 + num2;
	else if (word == '*')
		cout << num1 * num2;
	else if (word == '/')
		cout << num1 / num2;
	else
		cout << "Введён не правильный матиматический знак" << endl;

	system("pause>nul");
	return 0;
}